diff --git a/crates/recorder/src/measure/cycles.rs b/crates/recorder/src/measure/cycles.rs
index 7b8c7d6..64c37d3 100644
--- a/crates/recorder/src/measure/cycles.rs
+++ b/crates/recorder/src/measure/cycles.rs
@@ -12,7 +12,7 @@ lazy_static! {
         // CPU frequency, which adds ~5 seconds on start up time per
         // benchmarking process, and we only care about cycles anyways (less
         // affected by CPU frequency monitors).
-        let config = Config::default().wall_time(false);
+        let config = Config::default();
         Precision::new(config).unwrap()
     };
 }
@@ -39,5 +39,6 @@ impl Measure for CycleMeasure {
         let elapsed = end - self.0.take().expect("must call start before end");

         measurements.add(phase, "cycles".into(), elapsed.ticks());
+        measurements.add(phase, "nanoseconds".into(), elapsed.as_ns(precision));
     }
 }
